Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -94,37 +103,180 @@ private double calcCharityTrendingScore(Charity charity) {
             avgReview = userRating;
         }
         //TODO: Integrate db with correct method to retreive tags for one charity
```

Comment:
is this still valid?

Reply:
yes, in the case that a charity does not have a Charity Navigator rating, the avgReview equals the stored userRating per the algorithm in the design doc.